src/Generators/ControllerTagGenerator.php

```diff
@@ -39,6 +42,26 @@ protected function generateControllerObjectTags()
             if ($pageClassname !== 'Page') {
                 $this->pushMixinTag($pageClassname);
             }
+        } else if ($this->isContentController($this->className)) {
+            if (empty(self::$pageClassesCache)) {
+                self::$pageClassesCache = ClassInfo::subclassesFor(Page::class);
```

Comment:
We want to support usage on SS versions where the CMS is not installed. I guess the `isContentController` check above would stop this logic from running and causing an error if `Page` is undefined, is that right?

Reply:
Yes it would, and I just tried it on a project that does not have cms installed or any class called Page and it does seem to work without any crashes.